test/test_untilTag.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const dcmjs = require('C:/Users/DHDESK/Desktop/dcmjs/dcmjs/build/dcmjs.js')
+const buffer = fs.readFileSync('sample-dicom.dcm');
```

Comment:
Have a look at how [other tests](https://github.com/dcmjs-org/dcmjs/blob/master/test/test_data.js#L156) access data from the [data repository](https://github.com/dcmjs-org/data) to get sample data.

Reply:
I've written the integrated test in `test_data.js` to access the arbitrary `sample-dicom.dcm` the same way as other tests that access .dcms in the test folder.

Reply:
Thanks for the changes, it looks good now.